packages/components/psammead-bulletin/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-bulletin",
-  "version": "3.0.0",
+  "version": "4.0.0",
```

Comment:
I think this can be a patch change instead of a major change. This is since the only change happening here is a reordering of visually hidden and visible spans that are aria hidden, which should have no impact to using this component.

Reply:
The `LiveLabel` component hasn't been swarmed yet, so if we just bump the patch version, there will be the risk that someone else bumps it in Simorgh without being tested and integrate it, right? @sareh

Reply:
That's right. If we accessibility swarm this PR, then we should be able to have the live-label component brought out of alpha and imported into this PR.